src/dotnet/UI.Blazor/Components/Modal/BlazoredModal/Services/IModalService.cs

```diff
@@ -0,0 +1,79 @@
+﻿using Microsoft.AspNetCore.Components;
```

Comment:
Or maybe even combine this thing with `CascadingBlazoredModal`, remove the interface (i.e. keep just the impl.) & call it `BlazoredModalUI`.

Reply:
Notice there is "19 hidden conversations Load more…" thing that hides most of my comments ;)

Reply:
Removed the interface.